_docs/components/index.md

```diff
@@ -10,7 +10,38 @@ lead: See <a href="https://standards.usa.gov/alerts/">US Web Design Standards</a
 
   <div class="usa-contextual usa-contextual">
     <div class="usa-contextual-body">
+      <h1 class="usa-contextual-heading">Contextual Help and Support</h1>
+
+        <div class="usa-contextual usa-contextual">
+    <div class="usa-contextual-body">
+      <h2 class="usa-heading">Make it Simple</h2>
+      <p class="usa-contextual-text">Create a simplistic straight-forward user experience for Contextual Help or Support.</p>
+    </div>
+  </div>
+
+          <div class="usa-contextual usa-contextual">
+    <div class="usa-contextual-body">
+      <h2 class="usa-heading">Workflow for Complex Actions</h2>
+      <p class="usa-contextual-text">A decision based way for a user to choose a single or multiple actions in a process flow and in return lead to a result.</p>
+    </div>
+  </div>
+
       <h1 class="usa-contextual-heading">Contextual Help</h1>
+  
+  <div class="usa-contextual usa-contextual">
+    <div class="usa-contextual-body">
+     <h2 class="usa-heading">Inline Help - Help Text</h2>
+     <p class="usa-contextual-text">When providing help text display text to input field, like example below, which is a predictable location for users. </p>
+     </div>
+    </div>
+  <div class="preview">
+  <div>
+    <label for="input-type-text">Text input label</label>
+    <span class="usa-form-hint">Instructional text.</span>
+    <input id="input-type-text" name="input-type-text" type="text">
+  </div>
+  </div>
+
       <h2 class="usa-heading">Instructional Text</h2>
       <p class="usa-contextual-text">
       <p>Please set Instructional Text at a 192 Word Character Count (two line limit). These are the circumstances you use instructional text:</p>
```

Comment:
Can we move the footnote reference to just after the colon.

...instructional text:[footnote reference]

Then put the list inside a <blockquote>